src/Features/LanguageServer/Protocol/Extensions/ProtocolConversions.cs

```diff
@@ -612,6 +613,67 @@ public static ProjectId ProjectContextToProjectId(ProjectContext projectContext)
             return updatedOptions;
         }
 
+        public static LSP.MarkupContent GetDocumentationMarkdown(ImmutableArray<TaggedText> tags)
+        {
+            var builder = new StringBuilder();
+            var isInCodeBlock = false;
+            foreach (var taggedText in tags)
+            {
+                switch (taggedText.Tag)
+                {
+                    case TextTags.CodeBlockStart:
+                        builder.Append($"```csharp{Environment.NewLine}");
+                        builder.Append(taggedText.Text);
+                        isInCodeBlock = true;
```

Comment:
what about VB?  shouldn't this be based on the final language we're rending?

Reply:
Good catch, yeah this should be based on the document language